replica.go

```diff
@@ -31,8 +33,20 @@ type Replica struct {
 	isByz     bool
 	bElectNo  int
 	totalView int
-	timer     *time.Timer
-	eventChan chan interface{}
+
+	hasher     crypto.Hasher
+	signer     string
+	privateKey *crypto.PrivateKey
+	publicKeys []crypto.PublicKey
```

Comment:
I feel that it would be better to keep all the crypto stuff within the `crypto` package so that replica is not aware of keys and how to sign and verify